config/resque-pool.yml

```diff
@@ -1,3 +1,4 @@
+'*': <%= WORKER_CONCURRENCY %>
```

Comment:
Hm, we are changing our configuration for what workers we run?  Previously we had our configuration below, under `production`, we had various workers running on various queues. 

Now we also have this "*" key for all environments, saying to run default 5 workers that work on ALL queues. But we may ALSO be running those additional workers under `*`?

I don't think this is quite what we want to do. 

1. We should keep all our config under the `production` key
2. This change would change how many workers are running in our current production and staging environments if merged, we don't want to do that. We need a PR that allows us to keep things runnig as they are on current environment, but also if necessary allows us to set configuration to run them differently on heroku. 
2. Ideally, we could keep running the same configuration we have -- if you want to try different counts of workers, note you can already do that with no code change, just set heroku config vars `REGULAR_JOB_WORKER_COUNT`, `ON_DEMAND_JOB_WORKER_COUNT`, and `SPECIAL_JOB_WORKER_COUNT`.  Because our `ScihistDigicoll::Env` already uses the ENV variables that heroku config sets. You can set any of these to 0 (the last one already defaults to 0). 
3. If you DO need, at least for initial experimentation, to have a setup where you have 5 workers working all queues on heroku, instead of the setup we do have -- add a new line for `*` _under_ `production` area, which uses a *new( `ScihistDigicoll::Env` key you create, perhaps `ALL_QUEUE_JOB_WORKER_COUNT`. We can make sure it's set to 0 on our current infrastructure, but 5 on heroku, and you can set all the others to 0 on heroku. 

So, we need a code change that via config lets us keep our worker configuration the same on our existing infrastructure, but allows us to experiment with different things on heroku. But if we aren't going to end up with the same config on heroku eventually, we need to think through the ramifications.

Reply:
Yup -- will fix this. (This was me just copying and pasting without regard to the existing setup -- we definitely won't merge this as is.)

Reply:
Absolutely no need for this line; option 3 above works perfectly.

Reply:
Cool. I think it's possible you won't be able to fit more than TWO workers total in a standard-2x, once they get busy... but that's what we'll find out from testing! I can show you various ways to look at RAM usage on heroku that I have discovered, to figure out how much RAM my stuff is using, if I'm running out, etc.